addon/objects/uml-primitives/fd-uml-signal-receipt.js

```diff
@@ -0,0 +1,94 @@
+/**
+  @module ember-flexberry-designer
+*/
+
+import Ember from 'ember';
+
+import { BaseObject } from './fd-uml-baseobject';
+import FdUmlElement from './fd-uml-element';
+
+/**
+  An object that describes a Signal Reciept element on the UML diagram.
+
+  @class FdUmlSignalReceipt
+  @extends FdUmlElement
+*/
+export default FdUmlElement.extend({
+
+  /**
+    Signal Receipt name (actually its text).
+
+    @property name
+    @type String
+  */
+  name: Ember.computed.alias('primitive.Name.Text'),
+
+  /**
+    Type of primitive.
+
+    @property type
+    @type String
+  */
+  type: Ember.computed.alias('primitive.$type'),
+
+  /**
+    See {{#crossLink "FdUmlPrimitive/JointJS:method"}}here{{/crossLink}}.
+
+    @method JointJS
+  */
+  JointJS() {
+    let properties = this.getProperties('id', 'name', 'size', 'position', 'type');
```

Comment:
The `type` property must be obtained separately from the others, since it is not used in creating instances of classes for `JointJS`, also [here](https://github.com/Flexberry/ember-flexberry-designer/pull/189/files#diff-8ee60ff7cd0c76a16344ba45a81da637R40).